dxapp.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "verify_bam_id_v1.0",
+  "title": "verifyBamID_v1.0",
+  "summary": "v1.0 - Detects sample contamination from genotypic profiles; forked from moka-guys v1.1.1",
+  "tags": [
+    "Read QC",
+    "Statistics"
```

Comment:
Would be useful to have a shared set of these codes somewhere (or do these already exist in dnanexus?) so that we all tag apps in the same way (i.e. Statistics or Stats, not both)

Reply:
See comment below about confluence article.